doc/nrf/releases_and_maturity/releases/release-notes-2.7.0.rst

```diff
@@ -0,0 +1,1238 @@
+See :ref:`cloning_the_repositories` and :ref:`gs_updating_repos_examples` for more information.
+
+For information on the included repositories and revisions, see `Repositories and revisions for v2.7.0`_.
+
+IDE and tool support
+********************
+
+`nRF Connect extension for Visual Studio Code <nRF Connect for Visual Studio Code_>`_ is the recommended IDE for |NCS| v2.7.0.
+See the :ref:`installation` section for more information about supported operating systems and toolchain.
+
+Supported modem firmware
+************************
+
+See `Modem firmware compatibility matrix`_ for an overview of which modem firmware versions have been tested with this version of the |NCS|.
+
+Use the latest version of the nRF Programmer app of `nRF Connect for Desktop`_ to update the modem firmware.
+See :ref:`nrf9160_gs_updating_fw_modem` for instructions.
+
+Modem-related libraries and versions
+====================================
+
+.. list-table:: Modem-related libraries and versions
+   :widths: 15 10
+   :header-rows: 1
+
+   * - Library name
+     - Version information
+   * - Modem library
+     - `Changelog <Modem library changelog for v2.7.0_>`_
+   * - LwM2M carrier library
+     - `Changelog <LwM2M carrier library changelog for v2.7.0_>`_
+
+Known issues
+************
+
+Known issues are only tracked for the latest official release.
+See `known issues for nRF Connect SDK v2.7.0`_ for the list of issues valid for the latest release.
+
+Migration notes
+***************
+
+See the `Migration guide for nRF Connect SDK v2.7.0`_ for the changes required or recommended when migrating your application from |NCS| v2.6.0 to |NCS| v2.7.0.
+
+.. _ncs_release_notes_270_changelog:
+
+Changelog
+*********
+
+The following sections provide detailed lists of changes by component.
+
+Build and configuration system
+==============================
+
+* Added:
+
+  * Documentation section about the :ref:`file suffix feature from Zephyr <app_build_file_suffixes>` with a related information in the :ref:`migration guide <migration_2.7_recommended>`.
+  * Documentation section about :ref:`app_build_snippets`.
+  * Documentation section about :ref:`configuration_system_overview_sysbuild` and :ref:`sysbuild_enabled_ncs`.
+
+* Updated:
+
+  * All board targets for Zephyr's :ref:`Hardware Model v2 <zephyr:hw_model_v2>`, with additional information added on the :ref:`app_boards_names` page.
+  * The use of :ref:`cmake_options` to specify the image when building with :ref:`configuration_system_overview_sysbuild`.
+    If not specified, the option will be added to all images.
+  * The format version of the :file:`dfu_application.zip` files generated by the |NCS| build system.
+
+    Introducing Zephyr's :ref:`Hardware Model v2 <zephyr:hw_model_v2>` and building with :ref:`configuration_system_overview_sysbuild` for the |NCS| v2.7.0 release modified the content of the generated zip file. For example, binary file names and the board field were updated.
```

Comment:
Should we split to to have only one sentence per line?

Reply:
yes yes, thanks